src/requirements/requirement-exam-utils.ts

```diff
@@ -1,5 +1,5 @@
 import examData, { ExamFulfillment, ExamFulfillments } from './data/exams/ExamCredit';
-import { NO_EQUIVALENT_COURSES_COURSE_ID, CREDITS_COURSE_ID } from './data/constants';
+import { NO_FULFILLMENTS_COURSE_ID, CREDITS_COURSE_ID } from './data/constants';
 
 /** @deprecated old infra */
```

Comment:
What's the reasoning for keeping the deprecated code?

Reply:
So I can remove it all at once! That's step 5 😈 it will be glorious